docs/reference/getting-started.asciidoc

```diff
@@ -31,156 +31,158 @@ trial of Elasticsearch Service] in the cloud.
-
-[[getting-started-msi-installer-service]]
-image::images/msi_installer/msi_installer_no_service.png[]
-
-For configuration, simply leave the default values:
-
-[[getting-started-msi-installer-configuration]]
-image::images/msi_installer/msi_installer_configuration.png[]
-
-Again, to align with the tar example, uncheck all plugins to not install any plugins:
-
-[[getting-started-msi-installer-plugins]]
-image::images/msi_installer/msi_installer_plugins.png[]
-
-After clicking the install button, Elasticsearch will be installed:
-
-[[getting-started-msi-installer-success]]
-image::images/msi_installer/msi_installer_success.png[]
-
-By default, Elasticsearch will be installed at `%PROGRAMFILES%\Elastic\Elasticsearch`. Navigate here and go into the bin directory as follows:
-
-**with Command Prompt:**
-
-[source,sh]
++
+Windows:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
 cd %PROGRAMFILES%\Elastic\Elasticsearch\bin
+.\elasticsearch.exe
 --------------------------------------------------
++
+You now have a single-node {es} cluster up and running!
 
-**with PowerShell:**
-
-[source,powershell]
+. Start two more instances of {es} so you can see how a typical multi-node
+cluster behaves. You need to specify unique data and log paths
+for each node.
++
+Linux and macOS:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
-cd $env:PROGRAMFILES\Elastic\Elasticsearch\bin
+./elasticsearch -Epath.data=data2 -Epath.logs=log2
+./elasticsearch -Epath.data=data3 -Epath.logs=log3
 --------------------------------------------------
-
-And now we are ready to start our node and single cluster:
-
-[source,sh]
++
+Windows:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
-.\elasticsearch.exe
+.\elasticsearch.exe -Epath.data=data2 -Epath.logs=log2
+.\elasticsearch.exe -Epath.data=data3 -Epath.logs=log3
+--------------------------------------------------
++
+The default configuration enables the new nodes to automatically join the
+cluster with the first node because they are all running locally. Each node
+is automatically assigned a unique ID.
+
+. Use the `_cat/health` API to verify that your three node cluster is up running.
+The `cat` APIs return information about your cluster and indices in a
+format that's easier to read than raw JSON.
++
+You can interact directly with your cluster by submitting HTTP requests to
+the {es} REST API. Most of the examples in this guide enable you to copy the
+appropriate cURL command and submit the request to your local {es} instance from
+the command line. If you have Kibana installed and running, you can also
+open Kibana and submit requests through the Dev Console.
++
+TIP: When you're ready to start using {es} in your own applications, you'll
+want to check out the
+https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} language
+clients].
++
```

Comment:
I'd consider moving the "You can interact..." paragraph to the beginning of this list item.

Reply:
I opted to leave it to emphasize the step action.